plugins/woocommerce/client/admin/client/wp-admin-scripts/settings-embed/settings-ui.scss

```
@@ -64,12 +72,52 @@ body.woocommerce_page_wc-settings.woocommerce-settings-ui-page {
+		align-items: center;
+		border-radius: var(--wpds-border-radius-lg, 8px);
+		display: inline-flex;
+		font-size: var(--wpds-typography-font-size-sm, 12px);
+		font-weight: var(--wpds-typography-font-weight-regular, 400);
+		line-height: var(--wpds-typography-line-height-xs, 16px);
+		min-height: 24px;
+		padding: 0 var(--wpds-dimension-padding-sm, 8px);
+
+		&,
+		&--default {
+			background: var(--wpds-color-background-surface-neutral-weak, #f5f5f5);
```

Stylelint (17.14.1)

[error] The following CSS variables are not valid Design System tokens: '--wpds-color-background-surface-neutral-weak' (plugin-wpds/no-unknown-ds-tokens)

(plugin-wpds/no-unknown-ds-tokens)